makefiles/defaultmodules_late.inc.mk

```diff
@@ -0,0 +1,6 @@
+# This file declare DEFAULT_MODULEs that MAY only have as dependecies modules
```

Comment:
Some bikeshedding: The late vs. early is IMO more an implementation detail and from the name the implications are not obvious to users who are not familiar with the build system implementation details. IMO it is more about the "(potentially) with recursive deps" vs "guaranteed to not have recursive deps". And "can be deactivated during dependency resolution" vs "can't be deactivated during dependency resolution. I'm bad with names and cannot come up with a good name that makes it obvious that `defaultmodules_early.inc.mk` are the "regular" place to but them, while putting them into `defaultmodules_lates.inc.mk` requires special attention to guarantee that the do not have recursive deps. (My bad name suggestion would be `defaultmodules_regular.inc.mk` and `defaultmodules_no_recursive_deps.inc.mk`.)

Reply:
Long descriptive names are not an issue IMO!